### DISCUSSION TITLE
Download CSV of "Top Pages"

### DISCUSSION THREAD

**Discussion by jvns** [plausible/analytics#997] [Anything Else]
In the dashboard there's a download link for the time series at the top (`visitors.csv`).

It would be nice to be able to download a CSV with every page and how many time it's been visited from the dashboard. (I believe getting this data is possible with the API already)

**Comment by ukutaht** [comment]
Sounds good. I suppose this could be expanded to 'provide a csv export for all reports on the dashboard'

**Comment by metmarkosaric** [comment]
You can now export your top pages in a CSV format. Click on the download icon on the top right of your Plausible dashboard. It will export a ZIP file which includes CSV files of the individual reports. You can then import your stats into Excel or other data analysis tools. Learn more here: https://plausible.io/docs/export-stats
